### PR TITLE
[13.0] shopfloor misc improvements

### PR DESCRIPTION
**Pull request by simahawk**
Several fixes/imp, see atomic commits for details.
Most changes namely:
* zone picking add counters per each zone
* zone picking improve zone list display
* zone picking make sure you get always fresh data when navigating back
* show move lines / positions counters per each menu item
* upgrade vuetify

ref: 1780

### CODE REVIEW

**Review by TDu — APPROVED** [review]
I only skim-read this...

File: shopfloor/services/zone_picking.py [hunk 1]
```diff
@@ -249,9 +242,7 @@ def _data_for_location(self, zone_location, picking_type, location):
         }
 
     def _zone_lines(self, zones):
-        return self.env["stock.move.line"].search(
-            self._find_location_move_lines_domain(zones)
-        )
+        return self._find_location_move_lines(zones)
```
Comment (sebalix) [inline shopfloor/services/zone_picking.py:245]: Can't we do the following 
```suggestion
        return self.search_move_line.search_move_lines_by_location(zones)
```
and remove `_find_location_move_lines` method (which seems also used at different places but could be replaced by the component)? Even the `_zone_lines` method could be removed at this stage.

Comment (simahawk) [reply shopfloor/services/zone_picking.py:245]: I left it on purpose: if you want to completely customize the behavior at scenario level.

Comment (simahawk) [reply shopfloor/services/zone_picking.py:245]: @sebalix beside this the rest is good?

Comment (sebalix) [reply shopfloor/services/zone_picking.py:245]: gut!

**Review by sebalix — APPROVED** [review]

**Pull request merged**